simplq/src/components/Routes.jsx

```diff
@@ -14,10 +13,8 @@ export default () => {
       <ErrorBoundary>
         <Switch>
           <Route path="/" exact component={Home} />
-          <Route path="/create" exact component={CreateQueue} />
           <Route path="/queue/:queueId" exact component={AdminQueue} />
           <Route path="/j/:queueId" exact component={JoinQueueWithDetails} />
-          <Route path="/join" exact component={JoinQueueWithLink} />
           <Route path="/token/:tokenId" exact component={QueueStatus} />
```

Comment:
If joinQueueWithLink is not needed anymore, could you remove it?

Reply:
I've just now removed it in the next PR, where I have implemented the join queue button click

Reply:
Actually you are right, a separate component will be a good idea. I moved it out

https://github.com/SimplQ/simplQ-frontend/pull/218/commits/d7d953c83795a7d3c4e797aab8b04a69c37cf8c4